plugins/steve/resource-instance.js

```diff
@@ -0,0 +1,1645 @@
+import compact from 'lodash/compact';
```

Comment:
`resource-instance` now doesn't exist, you'll need to port changes to `resource-class`. Beat in mind also anything Steve/Norman specific may be better placed in

Reply:
Fixed :)